lib/evmone/eof.hpp

```diff
@@ -0,0 +1,26 @@
+// evmone: Fast Ethereum Virtual Machine implementation
+// Copyright 2020 The evmone Authors.
+// SPDX-License-Identifier: Apache-2.0
+#pragma once
+
+#include <stddef.h>
+#include <cstdint>
+
+namespace evmone
+{
+struct EOF1Header
+{
+    int code_size = 0;
+    int data_size = 0;
```

Comment:
Do we need to expose `data_size` now? It is not used for anything in the VM. Perhaps just leave a comment that this only cares about code?

Reply:
Currently it's used to find code section offset (i.e. total header size).